pages/knowledge/colours-and-contrast/how-to-examine/README.md

```diff
@@ -0,0 +1,43 @@
+---
+navigation_title: "How to examine"
+position: 2
+changed: "2019-04-21"
+---
+
+# How to examine colour contrast
+
+**Calculating contrast ratios is complex. Fortunately there are contrast calculators which take care of the math. We only need to provide the correct colour codes – which has a few potential pitfalls.**
+
+## Colour Contrast Analyser
+
+There are a lot of tools available to calculate the contrast between two colours. For Windows and macOS users we recommend the [Colour Contrast Analyser (CCA)](/setup/helper-tools/colour-contrast-analyser/).
+
+When calculating a contrast ratio, it is very important to use the “right” colour codes. In the context of web applications these are the values defined in CSS. But what is the best way to get to these values?
+
+## DOM inspector
+
+The most trustworthy source is the browser’s DOM inspector which gives direct access to the values defined in the source code.
+
+Right-click on an element and choose `Inspect` (depending on your browser the term might vary) to show that element in the DOM inspector. Now you can find the element’s colour values in the `Styles` tab.
```

Comment:
Instead of `depending on your browser the term might vary` I would write `the term might vary depending on your browser`.